packages/AlgernonTests.package/ALGTestTypes.class/instance/testSubmenuCheckBoxes.st

```diff
@@ -0,0 +1,15 @@
+testing world menu types
+testSubmenuCheckBoxes
+
+	| colorfulWindowsName colorfulWindowsInitialValue|
+	
+	colorfulWindowsName := 'Colorful Windows'.
+	colorfulWindowsInitialValue :=  Model useColorfulWindows.
+	self runQuery: colorfulWindowsName.
+	self assert: (self resultsIncludeName: colorfulWindowsName).
+	self doActionForItem: (self items first).
+	self deny: colorfulWindowsInitialValue equals: Model useColorfulWindows.
+	
+	Model useColorfulWindows: colorfulWindowsInitialValue
+	
+	
```

Comment:
if we happen to touch this method again, we should remove the trailing empty lines, it's not a must for now though (this is in other test methods too)